.github/workflows/main.yml

```diff
@@ -61,11 +45,11 @@ jobs:
     - name: Run pytest unittests on ubuntu
       if: matrix.os == 'ubuntu-latest'
```

Comment:
this `if` statement no longer needed if only testing on ubuntu

Reply:
.. oversight on my part...